CommentAgent.py

```diff
@@ -170,7 +170,9 @@ def parse_python_code(self, llm_answer):
             print("Found Python Block: ", python_code_block)
             return python_code_block
         else:
-            raise ParsingError("No python snippet provided")
+            raise ParsingError(
+                "No python snippet provided, remember to use ```python\n<valid_python_code>\n```"
```

Comment:
Remember that detailed error messages help developers understand what they need to fix. Consider including not just the instruction but also the reason why the code block is necessary.
```suggestion
        raise ParsingError(
"No python snippet provided. Please include a python code block using the markdown format ```python ... ``` for clarity and proper parsing."
        )
```